From 367c414e678c8bd0752e8c3c86a360f1a8c4247f Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Mon, 19 Jun 2023 09:38:43 +0200
Subject: [PATCH 0404/1141] drm/stm: lvds: set timout to 200ms

Reduce the pll time out from 20s to 200ms.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I1f574e30ad10e4693612c6b3c8980be644ffa6fd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/313525
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/gpu/drm/stm/lvds.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/stm/lvds.c
+++ b/drivers/gpu/drm/stm/lvds.c
@@ -107,7 +107,7 @@
 
 /* Sleep & timeout for pll lock/unlock */
 #define SLEEP_US	1000
-#define TIMEOUT_US	20000000
+#define TIMEOUT_US	200000
 
 #define PHY_SLV_OFS	0x100
 
